dojo/forms.py

```diff
@@ -1571,9 +1571,18 @@ def __init__(self, *args, **kwargs):
             finding = kwargs.pop('finding')
 
         super(ReviewFindingForm, self).__init__(*args, **kwargs)
+        self.fields['reviewers'].choices = self._get_choices(Dojo_User.objects.filter(is_active=True))
 
         if finding is not None:
-            self.fields['reviewers'].queryset = get_authorized_users_for_product_and_product_type(None, finding.test.engagement.product, Permissions.Finding_Edit)
+            queryset = get_authorized_users_for_product_and_product_type(None, finding.test.engagement.product, Permissions.Finding_Edit)
+            self.fields['reviewers'].choices = self._get_choices(queryset)
+
+    @staticmethod
+    def _get_choices(queryset):
+        l_choices = []
+        for item in queryset:
+            l_choices.append((item.pk, Dojo_User.generate_full_name(item)))
```

Comment:
should be `item.get_full_name()`

Reply:
But `item` is type  `'django.contrib.auth.models.User'` not  proxy class `Dojo_User`  and actually it is calling original method that is formed as `First_Name  Last_Name` (missing username).

So this will not work out of box (harmonized across other forms).

I assume then `get_authorized_users_for_product_and_product_type` should be reworked first https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/user/queries.py#L25

Reply:
I adapted `get_authorized_users_for_product_and_product_type` now it should be ok, I hope it will not break anything, but not results (format) looks fine.

Reply:
@StefanFl can you please take a look because it  touches `get_authorized_users_for_product_and_product_type`

Reply:
Good point @dsever . I think in general it's best if we return the `Dojo_User`  from our functions as it will have provide the extra abilities we sometimes want/need.